js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -74,8 +74,6 @@ type Props = {
   options: GraphExplorerOptions;
   setOptions?: (options: GraphExplorerOptions) => void;
 
-  assetFilterState?: AssetFilterState;
```

Comment:
oh could we just make this non optional? I think the prop not being passed in is probably a bug

Reply:
Discussed in Slack, it looks like the only scoping that happens above this component is passed in via `fetchOptions={{pipelineSelector}}`(on job pages) and all the other filter state is now within the AssetGraphExplorer, so it's ok to rm this!